actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   `ActionDispatch::Http::UploadedFile` now accepts `content_type` and `headers` when initializing
```

Comment:
<img width="443" alt="Screen Shot 2019-12-18 at 23 08 14" src="https://user-images.githubusercontent.com/350807/71127370-46f95d00-21eb-11ea-85ef-7c3e8fd7effd.png">

Does the `100644 → 100755` mean that we're changing file permissions? If so, let's not. Seems like they're on every file too.

Reply:
Also needs a period at the end of the title to fit with our changelog style.